packages/ag-charts-enterprise/src/features/color-picker/colorPicker.ts

```diff
@@ -91,10 +91,11 @@ export class ColorPicker extends AnchoredPopover<ColorPickerOptions> {
                 colorInput.value = colorString.toUpperCase();
             }
 
+            if (!trackChange) return;
+
+            this.hasChanged = true;
             const plainColor = Color.fromHSB(h, s, v, 1).toHexString();
             opts.onChange?.(colorString, plainColor, a);
-
-            if (trackChange) this.hasChanged = true;
```

Comment:
Previously, this would cause a ghost action to the history manager for undo/redo.